src/global_logic.js

```diff
@@ -958,12 +950,12 @@ module.exports.addSkilldataToTotals = function (totals, comb, arml, buff) {
 
     var index = 0;
     for (var key in totals) {
-        index = (index + 1) | 0;
+        index = index + 1 | 0;
         var isBahaAtIncluded = false;
         var isBahaAthpIncluded = false;
         var isBahaHpIncluded = false;
 
-        // オメガウェポンは別スキルなら複数効果発動可能
+        // Omega Weapon can activate multiple effects if it is different skill
```

Comment:
translate "Omega weapon" is ok for English ?
I note this
Japanese: Magna, Omega are 
English: Omega, Ultima (https://gbf.wiki/Category:Omega_Weapons  and  https://gbf.wiki/Ultima_Weapons)
in Motocal source code, magna and omega are used.

Reply:
I like omega here more because if I search parts in the logic I mostly search via the skill prefix in the skill list and that is baha, cosmos and omega. So it is just easier to find those parts when they have similar names.